graphdatascience/graph_data_science.py

```diff
@@ -39,7 +35,6 @@ def __init__(
         arrow: bool = True,
         arrow_disable_server_verification: bool = True,
         arrow_tls_root_certs: Optional[bytes] = None,
-        aura_db_connection_info: Optional[AuraDbConnectionInfo] = None,
```

Comment:
I like removing this. Technically it is breaking, but nobody could've used this for anything valuable yet anyway. The mistake is not removing it now, but adding it previously and releasing that.